tests/functional/firefox/test_android.py

```diff
@@ -15,6 +16,30 @@ def test_family_navigation(base_url, selenium):
     assert page.family_navigation.is_menu_displayed
 
 
+@pytest.mark.flaky(reruns=1)
```

Comment:
Do we have evidence that this is flaky or is it just using the same feature as the tests already marked as flaky?